src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -533,16 +519,22 @@ h2::before {
   }
 }
 
+#dissolution-date-section .v-card {
+  // remove extra space taken by error message
+  :deep(.v-text-field__details) {
+    margin-bottom: -8px !important;
+  }
+}
```

Comment:
Fixed for consistent inner border, ie:

![image](https://user-images.githubusercontent.com/10002889/193325288-28ec2f37-c53e-46cd-81ab-db51bbb0f6e5.png)